StickyTabBarViewController/Classes/StickyViewControllerSupportingTabBarController.swift

```diff
@@ -104,4 +103,16 @@ open class StickyViewControllerSupportingTabBarController: UITabBarController, S
         }
         collapsableVCFlow.expand()
     }
+    
+    private func removeStickyViewController() {
+        guard let childViewController = childViewController, let collapsableVCFlow = collapsableVCFlow else {
+            return
+        }
+        childViewController.view.removeFromSuperview()
+        childViewController.removeFromParent()
```

Comment:
Why do we need to make this steps as if `childViewController` was added as a child? 
As far as I can see this is only retained as a property but not added as a child.

Or am I missing something? 🤔

Reply:
Ah good catch, there is no need to call these methods on `childViewController`. 👍

Reply:
https://github.com/emrepun/StickyTabBarViewController/pull/17/commits/c8fa01c3d396936bdf2dcb9dddbe498c08a79a80 @Thurman1776-